Makefile

```diff
@@ -42,6 +42,9 @@ export AWS_ACCESS_KEY_ID ?=
 export AWS_SECRET_ACCESS_KEY ?=
 export AWS_SESSION_TOKEN ?=
 
+AWS_BUCKET_NAME ?= "kudo-ds-ci-artifacts"
+AWS_BUCKET_PATH ?= "/autodelete7d/spark-operator-history-server-test/"
```

Comment:
Not a big issue, but I think the leading and trailing slashes should be trimmed and concatenation should happen in the code. A possible alternative could be providing a comment describing the format of this property. BTW, is trailing slash mandatory?

Reply:
Sounds good, added concatenation to the code, and also removed the trailing slash